docs/step-by-step/kubeapps-on-tkg/introduction.md

```diff
@@ -0,0 +1,53 @@
+# Deploy and Configure Kubeapps on VMware Tanzu™ Kubernetes Grid
```

Comment:
I'm not sure, but, if we don't have any readme.md, Github won't include it in the folder previsualization. Let me explain:

https://github.com/kubeapps/kubeapps/tree/master/docs/user --> no readme, no "welcome file, no preview
https://github.com/kubeapps/kubeapps/tree/master/docs/developer --> a readme is displayed

It will depend on where do we want to put the docs, but as long as they are in gihthub, I find useful to have as a "landing" page. 

What do you think? Do you know any other alternatives?

Reply:
Added a README with bare introduction and link. I prefer to have the actual content separated from the README as it gives more flexibility for reuse in future.